ui/packages/registry-admin-remote-ui/src/main/webapp/less/styles.less

```diff
@@ -45,6 +45,10 @@ table.align-middle {
     color: red;
     display: block;
   }
+  .popover-content {
+    overflow-wrap: break-word;
```

Comment:
Looks like putting both overflow-wrap and word-wrap is redundant. 

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
Alright, removed that property and retested, thank you!